lib/games_manager.rb

```diff
@@ -250,6 +228,16 @@ def lowest_scoring_visitor
     average_min[0]
   end
 
+  def highest_scoring_home
+    averages = total_home_goals.merge(total_home_games) do |team_id, goals, games|
+      (goals/games.to_f).round(2)
+    end
+    average_max = averages.max_by do |team_id, average|
+      average
+    end
+     average_max[0]
```

Comment:
a more compact method for things like this might be 

highest_average = averages.key(averages.values.max)

basically, it makes an array of all the values, picks the biggest value, and then looks through the original hash, to find a key that connects to that value, and then returns that key. 

it would make 4 lines, into 1. not a big deal. but if your interested, that's one way to do it.